scripts/utils.js

```diff
@@ -0,0 +1,32 @@
+export const morseCodeObject = {
+    A: ".-",
+    B: "-...",
+    C: "-.-.",
+    D: "-..",
+    E: ".",
+    F: "..-.",
+    G: "--.",
+    H: "....",
+    I: "..",
+    J: ".---",
+    K: "-.-",
+    L: ".-..",
+    M: "--",
+    N: "-.",
+    O: "---",
+    P: ".--.",
+    Q: "--.-",
+    R: ".-.",
+    S: "...",
+    T: "-",
+    U: "..-",
+    V: "...-",
+    W: ".--",
+    X: "-..-",
+    Y: "-.--",
+    Z: "--..",
```

Comment:
You could add a translation for the space character here and save a bit of logic in the translation functions
```js
" ": "/ "
```

Reply:
Edge case: 
 - how to deal with multiple spaces. Maybe trim string and/or string.replace